.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Test Notifications Docker Image
+
+env:
+  IMAGE_NAME: notifications
```

Comment:
I'm not convinced we need either of these env vars, the tests don't do anything AWS related, and the IMAGE_NAME doesn't seem to be used either 🤔 

Same goes for the other files I think 🤔

Reply:
Well, I'll remove them and we'll find out 😸

Reply:
Seems fine for pre-merge...